env0/resource_template.go

```diff
@@ -126,6 +126,11 @@ func resourceTemplate() *schema.Resource {
 				Default:       "false",
 				ConflictsWith: []string{"gitlab_project_id", "token_id", "github_installation_id"},
 			},
+			"bitbucket_client_key": {
+				Type:        schema.TypeString,
+				Description: "The bitbucket client key used for integration",
+				Optional:    true,
+			},
```

Comment:
also update the `ConflictsWith` of others
since it can't be bitbucket and gitlab for example